buildspec/windowsTests.yml

```diff
@@ -2,13 +2,13 @@ version: 0.2
 env:
     variables:
         AWS_TOOLKIT_TEST_NO_COLOR: '1'
+        VSCODE_TEST_VERSION: '1.46.0' # TODO: Remove when we resolve debugger issues with integration tests
 phases:
     install:
-        commands:
-            - |
-              if(-Not($Env:CODE_COV_TOKEN -eq $null)) {
-                  choco install -y --no-progress codecov
-              }
+        commands: if(-Not($Env:CODE_COV_TOKEN -eq $null)) {
+            choco install -y --no-progress codecov
+            }
+            }
```

Comment:
Gratuitous brace appears to have caused the tests to fail (were changes to this block necessary?

Reply:
Yeah IDK what happened here. Reverted it.